yarn-project/aztec.js/src/wallet/base_wallet.ts

```diff
@@ -83,6 +83,9 @@ export abstract class BaseWallet implements Wallet {
   getRegisteredAccountPublicKeysHash(address: AztecAddress): Promise<Fr | undefined> {
     return this.pxe.getRegisteredAccountPublicKeysHash(address);
   }
+  getRegisteredAccountPublicKeys(address: AztecAddress): Promise<Point[] | undefined> {
```

Comment:
My brain would prefer a struct instead of an array of points where we need to remember an ordering etc 🤷